docs/multicluster/antctl.md

```diff
@@ -36,57 +36,96 @@ To see the usage examples of these commands, you may also run `antctl mc get [su
 
-## antctl mc deploy
+## antctl mc join
 
-`antctl mc deploy` command can deploy Antrea Multi-cluster Controller to a leader or member cluster.
+`antctl mc join` command lets a member cluster join an existing Antrea Multi-cluster ClusterSet. It will create a
+ClusterSet and ClusterClaims for the member cluster. Users can use command line options or a config file (which can
+be the output file of the `anctl mc init` command) to specify the ClusterSet join arguments.
 
-+ `antctl mc deploy leadercluster` command can deploy Antrea Multi-cluster Controller to a leader cluster, and define all the CRDs the leader cluster needed.
-+ `antctl mc deploy membercluster` command can deploy Antrea Multi-cluster Controller to a member cluster, and define all the CRDs the member cluster needed.
+When the config file is provided, the command line options may be overwritten by the file. A token is needed for a
+member cluster to access the leader cluster API server. Users can either specify a pre-created token Secret with the
+`--token-secret-name` option, or pass a Secret manifest to create the Secret with either the `--token-secret-file`
+option or the config file.
 
 ```bash
-antctl mc deploy leadercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
-antctl mc deploy membercluster [--antrea-version ANTREA_VERSION] [-n NAMESPACE] [-f PATH_TO_MANIFEST]
+antctl mc join --clusterset=CLUSTERSET_ID \
+                   --clusterid=CLUSTER_ID \
+                   --namespace=MEMBER_NAMESPACE \
+                   --leader-clusterid=LEADER_CLUSTER_ID \
+                   --leader-namespace=LEADER_NAMESPACE \
+                   --leader-apiserver=LEADER_APISERVER \
+                   --token-secret-name=TOKEN_SECRET_NAME \
+                   --token-secret-file=TOKEN_SECRET_FILE
+
+antctl mc join --config-file PATH_TO_CONFIG_FILE
 ```
 
-To see the usage examples of these commands, you may also run `antctl mc deploy [subcommand] --help`.
+Below is a config file example:
+
+```yaml
+apiVersion: multicluster.antrea.io/v1alpha1
+kind: ClusterSetJoinConfig
+clusterSetID: clusterset1
+clusterID: cluster-east
+namespace: kube-system
+leaderClusterID: cluster-north
+leaderNamespace: antrea-multicluster
+leaderAPIServer: https://172.18.0.3:6443
+tokenSecretName: cluster-east-token
+```
+
+## antctl mc leave
+
+`antctl mc leave` command lets a member cluster leave a ClusterSet. It will delete the ClusterSet and ClusterClaims
+and other resources created by antctl for the member cluster.
+
+```bash
+antctl mc leave --clusterset CLUSTERSET_ID --namespace [NAMESPACE]
+```
+
+## antctl mc destroy
+
+`antctl mc destroy` command can destroy an Antrea Multi-cluster ClusterSet in a leader cluster. It will delete the
+ClusterSet and ClusterClaims and other resources created by antctl for the leader cluster.
+
+```bash
+antctl mc destroy --clusterset=CLUSTERSET_ID --namespace [NAMESPACE]
```

Comment:
Namespace is not optional for "destroy".

Reply:
Removed the "[]"